quality_indicators/indicator_schema.json

```diff
@@ -0,0 +1,241 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://everse.software/indicator_schema.json",
+    "type": "object",
+    "default": {},
+    "title": "EVERSE indicator schema",
+    "required": [
+        "identifier",
+        "name",
+        "description",
+        "status",
+        "quality-dimension",
+        "release-date",
+        "doi",
+        "authors",
+        "contacts"
+    ],
+    "$defs": {
+        "person": {
```

Comment:
As discussed today, we decided to leave person and org out of the scope for now

Reply:
done